binfmt-misc/pkg.yaml

```diff
@@ -0,0 +1,23 @@
+name: binfmt-misc-pkg
```

Comment:
we could probably skip this, the module will be part of the `kernel` pkg this way, but not added by default to Talos initramfs (that's where the extension would help)

Reply:
@frezbo please correct me if I'm wrong - do we actually need to a separate package here in `pkgs`, or we could simply strip out a single file from `kernel` for the extension?

Reply:
hmm, good point, we can definitelty do that from extensions and drop shipping `-pkg` suffixes images for in-tree modules

Reply:
created #805 to track

Reply:
ok, sounds good, I'll update this PR, thanks